doc/man/s3fs.1.in

```diff
@@ -168,10 +168,24 @@ specify expire time (seconds) for entries in the stat cache and symbolic link ca
 specify expire time (seconds) for entries in the stat cache and symbolic link cache. This expire time is based on the time from the last access time of those cache.
 This option is exclusive with stat_cache_expire, and is left for compatibility with older versions.
 .TP
-\fB\-o\fR disable_noobj_cache (default is enable)
-By default s3fs memorizes when an object does not exist up until the stat cache timeout.
-This caching can cause staleness for applications.
-If disabled, s3fs will not memorize objects and may cause extra HeadObject requests and reduce performance.
+\fB\-o\fR enable_negative_cache (default is enabled negative cache)
+This option will keep non-existence of objects in a stat cache.
+When this negative cache is enabled, it will not process extra HeadObject requests to search for non-existent objects, improving performance.
+This feature is enabled by default, so there is no need to specify it.
+.TP
+\fB\-o\fR enable_noobj_cache (default is enabled negative cache)
```

Comment:
Just remove the negative option from the documentation.

Reply:
Ok, it will be removed.